src/code42cli/cmds/securitydata.py

```diff
@@ -253,9 +245,86 @@ def show(state, search_id):
     echo(pformat(response["searches"]))
 
 
+@security_data.command()
+@file_event_options
+@search_options
+@click.option(
+    "--or-query", is_flag=True, cls=searchopt.AdvancedQueryAndSavedSearchIncompatible
+)
+@sdk_options()
+@server_options
+def send_to(
+    state,
+    format,
+    hostname,
+    protocol,
+    begin,
+    end,
+    advanced_query,
+    use_checkpoint,
+    saved_search,
+    or_query,
+    **kwargs
+):
+    """Send events to the given server address."""
+    handlers = _extract_events(
+        state,
+        begin,
+        end,
+        advanced_query,
+        use_checkpoint,
+        saved_search,
+        or_query,
+        output_function=send_events,
+        **kwargs,
+    )
+    if not handlers.TOTAL_EVENTS and not errors.ERRORED:
+        echo("No results found.")
+    else:
+        output_func = handlers.output_func
+        output_func(format, hostname, protocol, SEARCH_DEFAULT_HEADER)
```

Comment:
I'm a little bit confused by this, I would think that outputting would be handled by `_extract_events`, especially since `send_events` is passed to it. Why does this get called here?

Reply:
I think `output_func` should be called from within `handle_response` -- unless I'm misunderstanding something, I think that this will only output the last page.

Reply:
Changed.